stock_picking_send_by_mail/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Sandra Figueroa Varela>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking by Mail",
+    "summary": "Send stock picking by email",
+    "version": "8.0.1.0.0",
+    "author": "<Sandra Figueroa Varela>, Odoo Community Association (OCA)",
+    "website": "",
```

Comment:
Please modify this entry following https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py template